File: app/src/main/java/com/lazylee/lzywanandroid/data/entity/Banner.java
package com.lazylee.lzywanandroid.data.entity;

import androidx.room.PrimaryKey;

/**
 * banner 实体类
 * Created by lazylee on 2018/3/23.
 */

public class Banner {
    @PrimaryKey
    private long id;
    private int order;
    private int type;
    private int isVisible;
    private String desc;
    private String imagePath;
    private String title;
    private String url;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public int getOrder() {
        return order;
    }

    public void setOrder(int order) {
        this.order = order;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public int getIsVisible() {
        return isVisible;
    }

    public void setIsVisible(int isVisible) {
        this.isVisible = isVisible;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getImagePath() {
        return imagePath;
    }

    public void setImagePath(String imagePath) {
        this.imagePath = imagePath;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    @Override
    public String toString() {
        return "Banner{" +
                "id=" + id +
                ", order=" + order +
                ", type=" + type +
                ", isVisible=" + isVisible +
                ", desc='" + desc + '\'' +
                ", imagePath='" + imagePath + '\'' +
                ", title='" + title + '\'' +
                ", url='" + url + '\'' +
                '}';
    }
}

File: app/src/main/java/com/lazylee/lzywanandroid/data/entity/Banner.kt
package com.lazylee.lzywanandroid.data.entity

import androidx.room.PrimaryKey

/**
 * banner 实体类
 * Created by lazylee on 2018/3/23.
 */

class Banner {
    @PrimaryKey
    var id: Long = 0
    var order: Int = 0
    var type: Int = 0
    var isVisible: Int = 0
    var desc: String? = null
    var imagePath: String? = null
    var title: String? = null
    var url: String? = null

    override fun toString(): String {
        return "Banner{" +
                "id=" + id +
                ", order=" + order +
                ", type=" + type +
                ", isVisible=" + isVisible +
                ", desc='" + desc + '\''.toString() +
                ", imagePath='" + imagePath + '\''.toString() +
                ", title='" + title + '\''.toString() +
                ", url='" + url + '\''.toString() +
                '}'.toString()
    }
}
